vscode/src/common.ts

```diff
@@ -3,6 +3,8 @@
 
 import { TextDocument, Uri, Range, Location } from "vscode";
 import { ILocation, IRange } from "qsharp-lang";
+import { IWorkspaceEdit } from "../../npm/qsharp/lib/web/qsc_wasm";
```

Comment:
Again, just import from the 'qsharp-lang' package (i.e. add to the imports in the line above). If you need to re-export more things in that package, then do so.